slimt/Annotation.hh

```diff
@@ -250,5 +273,19 @@ class AnnotatedText {
     return std::string_view(text.data() + range.begin, range.size());
   }
 };
+class WordIterator {
```

Comment:
nit: line between. 
larger question: Should we enable [`SeparateDefinitionBlocks: always`](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#separatedefinitionblocks)?